stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/MessageItem.kt

```diff
@@ -92,10 +93,11 @@ import io.getstream.chat.android.compose.ui.util.isUploading
  */
 @OptIn(ExperimentalFoundationApi::class)
 @Composable
-public fun MessageItem(
+public fun LazyItemScope.MessageItem(
     messageItem: MessageItemState,
     onLongItemClick: (Message) -> Unit,
-    modifier: Modifier = Modifier,
+    modifier: Modifier = Modifier
+        .animateItemPlacement(),
```

Comment:
As a result of the way we create functions according to the `DefaultComposables` template that do not expose a modifier we have to make all of our Composables that want to be animated in a LazyList an extension function of `LazyItemScope`.